service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisDocumentGeneratorService.java

```diff
@@ -15,48 +15,46 @@
 import uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates;
 import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
 import uk.gov.hmcts.reform.fpl.model.common.DocmosisRequest;
+import uk.gov.hmcts.reform.fpl.model.docmosis.AbstractDocmosisData;
 
 import java.util.Map;
 
 @Service
 @Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class DocmosisDocumentGeneratorService {
     private final RestTemplate restTemplate;
-    private final Logger logger = LoggerFactory.getLogger(getClass());
-    private final DocmosisConfiguration docmosisDocumentGenerationConfiguration;
-
-    @Autowired
-    DocmosisDocumentGeneratorService(RestTemplate restTemplate,
-                                     DocmosisConfiguration docmosisDocumentGenerationConfiguration) {
-        this.restTemplate = restTemplate;
-        this.docmosisDocumentGenerationConfiguration = docmosisDocumentGenerationConfiguration;
+    private final DocmosisConfiguration configuration;
+    private final ObjectMapper mapper;
+
+    public DocmosisDocument generatedDocmosisDocument(AbstractDocmosisData templateData, DocmosisTemplates template) {
```

Comment:
generate or generated? I think that generate makes more sense as that is what is actually happening.

Reply:
definitely generate, typo by me